pom.xml

```diff
@@ -634,6 +634,9 @@
                             <!-- Moving the interface methods to new SRA super interfaces causes japicmp to complain -->
                             <exclude>software.amazon.awssdk.auth.credentials.AwsCredentials</exclude>
                             <exclude>software.amazon.awssdk.auth.token.credentials.SdkToken</exclude>
+
+                            <!-- Changing isAnonymous() to use the new SRA super interfaces causes japicmp to complain -->
+                            <exclude>software.amazon.awssdk.auth.credentials.CredentialUtils</exclude>
```

Comment:
Yeah, it was right to complain since that's a protected class (see my comment above). Binary backwards-compatibility doesn't allow for changing parameter types to super-types or return types to sub-types, even though intuition would say it's "safe" to do.